build.gradle

```diff
@@ -12,17 +12,21 @@ release {
 
 group = "com.splunk.public"
 
+boolean snapshot = getProperty("snapshot") == 'yes'
```

Comment:
I thought `getProperty` throws an exception if no such property exists?

Reply:
As you can see in PR build - it doesn't.